packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -122,7 +122,7 @@ export class MockContainerRuntime {
         this.dataStoreRuntime.deltaManager = this.deltaManager;
         this.dataStoreRuntime.quorum = factory.quorum;
         // FluidDataStoreRuntime already creates a clientId, reuse that so they are in sync.
-        this.clientId = this.dataStoreRuntime.clientId;
+        this.clientId = this.dataStoreRuntime.clientId ?? uuid();
```

Comment:
MockFluidDataStoreRuntime.clientId is always assigned to uuid().  we should change the type on that.  

Again that is a breaking change, so it can only target the next branch.